composer.json

```diff
@@ -4,29 +4,35 @@
     "require": {
         "php": "^7.2",
         "ext-json": "*",
-        "beberlei/assert": "^2.9",
-        "composer/composer": "^1.6",
+        "beberlei/assert": "^3.2.0",
+        "composer/composer": "^1.8.5",
```

Comment:
any particular reason why you put this bar that high?

e.g. on travis we only have 1.8.0 and need a extra step to update to ^1.8.5

Reply:
@staabm this does not affect the version of composer you have installed on your machine. This is a requirement for installing composer itself as a dependency of this package (as a library)

Reply:
Besides that, I generally always bump all dependencies to their latest stable, unless there is a strong reason not to do so. This already solves the grand majority of issues for most users that may otherwise get an outdated dependency set (and removes trouble around "please upgrade before reporting an issue")